woocommerce/woocommerce-ios
# [React Native] Integrate Shipping Zones

# Why

This PR integrates the React Native library which allows us to view the store shipping zones.

# How

- Integrate the RN library via SPM
- Add a shipping zone row in the settings screen.
- Launch the RN library with the current store credentials.

# Demo

https://github.com/woocommerce/WooCommerce-Shared/assets/562080/4d944ea8-cdde-46b8-aff2-6e3b71049fa8

# Testing Steps

- Launch the app
- Go to the settings screen
- Tap on the Shipping Zones row
- See that the zones render correctly.

---
